backend/burning-okr/burning-okr-app/src/main/java/org/burningokr/controller/okr/TopicDraftController.java

```diff
@@ -161,4 +143,28 @@ public ResponseEntity deleteTopicDraftById(@PathVariable Long topicDraftId, User
     okrTopicDraftService.deleteTopicDraftById(topicDraftId, user);
     return ResponseEntity.ok().build();
   }
+
+  /**
+   * Endpoint to create a OkrUnit from a TopicDraft
+   *
+   * @param okrUnitId The Identifier for the OkrUnit underneath which the new Unit should be created
+   * @return The Identifier of the newly created Team
+   */
+  @GetMapping("/topicDraft/convertToTeam")
+  @PreAuthorize("@authorizationService.isAdmin()")
+  public ResponseEntity<OkrDepartmentDto> convertTopicDraftToTeam(
+      @RequestParam(name = "topicDraftId") long topicDraftId,
+      @RequestParam(name = "okrUnitId") long okrUnitId,
+      User user) {
+    Logger.getLogger("TopicDraftController")
```

Comment:
Ist etwas flexibler:
```suggestion
Logger.getLogger(TopicDraftController.class.getName())
```